moped-editor/src/views/projects/projectView/ProjectComponents/TheMap.js

```diff
@@ -111,7 +111,7 @@ export default function TheMap({
         type: "show_error",
         payload: { message: "Zoom in to select features", severity: "error" },
       });
-    } else {
+    } else if (isCreatingComponent || isEditingComponent) {
```

Comment:
This fixes a bug where there was competition between the zoom error alert and the "can't find linked component in this project" alert. Didn't really think about how multiple error alerts would work together when I wrote that original code.

Reply:
I can't tell how this would be related, but i am now seeing the loading spinner (CircularProgress) that typically shows during editing—while we're fetching AGOL features—when i am not editing.

To reproduce, edit a line feature and save your edits. zoom in/out a bit and you should see the spinner flashing in the toolbar.

I don't _think_ this is a pre-existing bug?

<img width="1090" alt="Screenshot 2023-10-18 at 2 24 51 PM" src="https://github.com/cityofaustin/atd-moped/assets/14793120/72d2dffc-b51e-4a3e-b7df-c673fe71189d">

Reply:
@johnclary wow - this is a bug in prod! I checked the code, and you can see [right here](https://github.com/cityofaustin/atd-moped/blob/19c5556c8d1ab3db9a733ccd2e89d736f75384d1/moped-editor/src/views/projects/projectView/ProjectComponents/utils/agol.js#L110C9-L110C9) that fetching lines starts happening when `linkMode` is set. After saving an edit, [I left out switching `linkMode` back to `null`](https://github.com/cityofaustin/atd-moped/blob/19c5556c8d1ab3db9a733ccd2e89d736f75384d1/moped-editor/src/views/projects/projectView/ProjectComponents/utils/useUpdateComponent.js#L401-L412) which does happen [after creating a component](https://github.com/cityofaustin/atd-moped/blob/19c5556c8d1ab3db9a733ccd2e89d736f75384d1/moped-editor/src/views/projects/projectView/ProjectComponents/utils/useCreateComponent.js#L155-L167). This is a one-liner so I'll patch it.

Reply:
whoaa—now i do see it in prod 😵 

thanks for stitching that up!!